CONTRIBUTING.md

```diff
@@ -1,17 +1,18 @@
-Esri welcomes contributions from anyone and everyone. Please see our
-[guidelines for contributing](https://github.com/esri/contributing).
+# Contribution guidelines
 
-# Contributing to demos for the ArcGIS Runtime SDK for .NET
+Esri welcomes contributions to our open source projects on Github. Please see our [guidelines for contributing](https://github.com/esri/contributing).
```

Comment:
In the context of the demos repo, should these Contributing guidelines set a more specific expectation?

e.g.
> We do not currently accept new PRs for full features, for example an entire new demo app or a significant change to an existing demo app, but we do encourage you to submit issues for:
> - Bugs you find while using the demos.
> - Requests for new demo apps or enhancements to existing demo apps.